crates/router/src/configs/settings.rs

```diff
@@ -420,6 +420,7 @@ pub struct Secrets {
 #[serde(default)]
 pub struct Locker {
     pub host: String,
+    pub host_rs: String,
     pub mock_locker: bool,
     pub basilisk_host: String,
     pub locker_signing_key_id: String,
```

Comment:
will all the keys be used simultaneously?

should this be an enum instead where u choose the type of locker you wanna use... (mock/rust/basilisk etc)

Reply:
we will be needing all the keys simultaneously, as we are having a fallback for locker as part of pr
https://github.com/juspay/hyperswitch/pull/2888